tests/driver_feetech/main.c

```diff
@@ -98,7 +98,7 @@ static int32_t parse_baud(char *arg)
     int32_t baud = (int32_t)atoi(arg);
 
     for (size_t i = 0 ; i < ARRAY_LEN(baudrates) ; i++) {
-        if(baud == baudrates[i]) {
+        if (baud == baudrates[i]) {
```

Comment:
I changed my mind : it will simpler if you provide those space changes in a separate commit in this PR.